docker-compose-mac.yml

```diff
@@ -123,4 +123,4 @@ services:
       editoast: {condition: service_healthy}
     restart: "no"
     image: alpine:latest
-    command: "true"
+    command: "true"
```

Comment:
Missing trailing line feed. Check how to configure it under mac: https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files

Reply:
Done